MANUAL.txt

```diff
@@ -1148,10 +1148,18 @@ Options affecting specific writers {.options}
     If used multiple times, the arguments are provided with spaces between
     them. Note that no check for duplicate options is done.
 
+`--latex-table-environment=longtable`|`tabularx`
+
+:   Use the specified table environment when producing LaTeX output.
+    The default is [`longtable`] which allows table to spread over multiple pages.
+    [`tabularx`] allows for paragraph-like columns whose width automatically expand to fill the width of the environment.
```

Comment:
Lines should be hard-wrapped to fit within 80 columns.

Reply:
I'm not sure I quite understand this description: "paragraph-like columns whose width automatically expand to fill the width of the environment."  Maybe better to say something about what should dictate this choice?  For example: If you have long tables that may need to spans multiple pages, use `longtable`.  If you want to create floating tables (and for better compatibility with other floating elements on the page), use `tabularx`.